src/plone/testing/zope.py

```diff
@@ -461,6 +460,11 @@ def setUpDatabase(self):
         that the database that is opened by Zope 2 is in fact the top of
         the resource stack.
         """
+
+        # NOTE: Keep that import here.
+        #       Having it on module-level has the side effect of injecting the
+        #       ZopeLite test configuration into the global Zope configuration.
+        from Testing.ZopeTestCase.ZopeLite import _patched as ZOPETESTCASEALERT
```

Comment:
I think we can remove this and the following lines all together?
Looking at it, I can't make sense of importing something and then throwing an error if the import was successful.

Reply:
I agree. It seems that running `ZopeTestCase` together with ours had side-effects that needed to be warned about. See https://github.com/plone/plone.testing/commit/3268cda0ecddab51dbe7bfc3c7c7b997777f9ac0 
I should hope that these issues are gone.

Reply:
Ah, yes, that makes sense.
I remeber also, that was done at the Alpine City Sprint.